home.html

```diff
@@ -0,0 +1,154 @@
+            <div class="bar"></div>
+            <div class="bar"></div>
+            <div class="bar"></div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </header>
+  <section>
+    <img src="./images/download (7).jpg" class="image" alt="img">
+  </section>
+
+  <section class="headline">
+    <div class="headline-container">
+      <p class="headline-p">"Hello! scholar"</p>
+      <h1>Educational Resources Summit 2023</h1>
+      <p class="headline-summary">
+        Educational resources are used in a learning environment to help and assist with people’s development and
+        learning. We are designed to reinforce learning and in some cases allow people to put their knowledge to the
+        test. Educational resources are brilliant for teachers and educators to help them deliver the best quality
+        lessons.
+      </p>
+      <h2>
+        2023.02.20(THU) ~ 21(FRI)
+      </h2>
+      <P class="venue">
+        @ Educational Resources center, ZONE 7 FCT Abuja Nigeria.
+      </P>
+    </div>
+  </section>
+  <section class="event" id="event">
+    <h3>Event Activities </h3>
+    <hr class="event-hr">
+    <div class="cards">
+      <div class="event-container">
+        <div class="event-box box1">
+          <img src="./images/download.jpg" class="event-img" alt="">
+          <h2 class="event-h2">Workshop</h2>
+          <p class="event-p">
+            inspiring intellectual curiosity & develop research skills & confidence in young minds.
+          </p>
+        </div>
+        <div class="event-box">
+          <img src="./images/download (4).jpg" class="event-img" alt="">
+          <h2 class="event-h2">Lecture</h2>
+          <p class="event-p">
+            listen to speakers from various countries about the messages of sharing and opening.
+          </p>
+        </div>
+        <div class="event-box">
+          <img src="./images/images.jpg" class="event-img" alt="img">
+          <h2 class="event-h2">Forum</h2>
+          <p class="event-p">
+            Access to share your thoughts amd opinions with expertsfor each topic.
+          </p>
+        </div>
+        <div class="event-box">
+          <img src="./images/download (6).jpg" class="event-img" alt="">
+          <h2 class="event-h2">Book Launch</h2>
+          <p class="event-p">
+            New inspiring books and also new versions of books will be published.
+          </p>
+        </div>
+      </div>
+    </div>
+    <button class="btn">Join Educational resources Summit 2023</button>
+  </section>
+  <section class="feature" id="guest">
+  </section>
+  <section class="partner" id="sponsor">
+    <h2>Sponsor</h2>
+    <div class="ul-container">
+      <ul class="partner-ul">
+        <li class="partner-li"> <em>Google</em> </li>
+        <li class="partner-li"><em>Mozilla</em></li>
+        <li class="partner-li"><em>naver</em></li>
+        <li class="partner-li"><em>Addidas</em></li>
+      </ul>
+    </div>
+  </section>
```

Comment:
- [ ] Kindly make sure you match the design of this section to the one of the template design
### Template design
![Screenshot from 2023-01-14 21-26-55](https://user-images.githubusercontent.com/83042742/212492697-4cfdf23b-7a0d-41c9-957b-6c274044f604.png)

### Your Design
![Screenshot from 2023-01-14 21-24-43](https://user-images.githubusercontent.com/83042742/212492704-ce91c32e-b3a0-47e2-bb8c-35358adf84e4.png)